src/components/form-elements/Input.spec.tsx

```diff
@@ -1,72 +1,81 @@
 import * as React from 'react';
 import Input from './Input';
-import {shallow} from 'enzyme';
+import {render} from '@testing-library/react';
 
 test('render', () => {
-  const component = shallow(<Input />);
-  const input = component.find('input');
+  const component = render(<Input />);
 
-  expect(input.hasClass('sg-input')).toEqual(true);
+  expect(component.queryByRole('textbox')).toBeTruthy();
```

Comment:
I would prefer to use `getBy` if we are looking to confirm presence of an element.